models/base/xapi_events_all_parsed.sql

```diff
@@ -9,34 +9,34 @@
 
 SELECT
     event_id as event_id,
-    JSON_VALUE(event_str, '$.verb.id') as verb_id,
+    JSON_VALUE(event::String, '$.verb.id') as verb_id,
```

Comment:
This cast is necessary for dbt to run these models against an empty table for some reason.